charts/spire/charts/spire-server/values.yaml

```
@@ -184,6 +183,25 @@ upstreamAuthority:
         certificate: ""
         key: ""
         bundle: ""
+  awsPCA:
+    enabled: true
```

Should default to off
```suggestion
    enabled: false
```